conf/cassandra.yaml

```diff
@@ -63,6 +63,11 @@ hinted_handoff_enabled: true
 # created until it has been seen alive and gone down again.
 max_hint_window_in_ms: 10800000 # 3 hours
 
+# The file size limit to store hints for an unreachable host.
+# Once the local hints files have reached the limit, no more new hints will be created.
+# Set a non-positive value will disable the size limit.
+max_hints_size_per_host_in_mb: 0
```

Comment:
should this be `max_hint_size_per_host_in_mb` instead, similar to `max_hint_window_in_ms`. So drop the plural to make it consistent?

Reply:
yeah, let's make it consistent with the majority

Reply:
Agree with @yifan-c, 'hints' reading better